pom.xml

```diff
@@ -563,6 +582,12 @@
       </dependency>
 
       <dependency>
+        <groupId>com.google.guava</groupId>
```

Comment:
What's this new dependency for?

Reply:
Aaah, had to remove it to remember, `io.swagger:swagger-jaxrs` requires Guava 20.0, let me see if I can remove `swagger-jaxrs` instead.

This is the original error:
```
[ERROR] Found a problem with the direct dependency com.google.guava:guava of the current project
  Expected version is 18.0
  Resolved version is 18.0
  Version 20.0 was expected by artifact: io.swagger:swagger-jaxrs
```

Reply:
Yeah, can't do that we're using it in the `rest` module (`io.syndesis.rest.v1.SwaggerConfiguration`)